dispatch/static/manager/src/js/pages/Events/EventAuditPage.js

```diff
@@ -75,6 +76,13 @@ class EventAuditPage extends React.Component {
     this.props.disapproveEvent(this.props.token, id)
   }
 
+  pagination() {
```

Comment:
I don't think this needs to be a function. A `const` in the render function should be sufficient.